portfolio/src/main/webapp/index.html

```diff
@@ -6,48 +6,76 @@
         <link rel="stylesheet" href="style.css">
     </head>
     <body>
-        <p> Preview a video</p>
-        <!-- Form for data servlet -->
-        <form id="video-data-form">
-            <!-- Should change to URL later -->
-            <label for="videoId">Id&nbsp;</label>
-            <input type="text" id="videoId" name="videoId"><br>
-        </form>
-        <button onclick="submitVideoData()">Load Video</button>
-        <div id = "video-box">
-            <p id = "video-title"></p>
-            <p id = "video-description"></p>
+        <!-- Section for searching for a video or playlist. -->
+        <div id="preview-search-flexbox">
```

Comment:
You could put this div inside the search section.

Reply:
Also this is a flexbox for the entire search area so it doesn't quite make sense to name it preview-search-flexbox.

Reply:
Nvm guess you meant preview and search?